cookbook/core/flyte_basics/task_cache.py

```diff
@@ -39,10 +39,11 @@ def square(n: int) -> int:
 # %%
 # If, in a subsequent code update, we update the signature of the task to return the original number along with the result, it'll automatically invalidate the cache (even though the cache version remains the same). ::
 #
-#   :py:func:`flytekit.task`
-#   @task(cache=True, cache_version="1.0")
-#   def square(n: int) -> (int, int):
-#       ...
+# :py:func:`flytekit.task`
```

Comment:
this was meant to be an example of how an update to the signature will look like

Reply:
@wild-endeavor wdyt?

Reply:
This wasn't clear, it seemed like it was just a mistake. We could maybe make it look like a diff? I'm not sure how though

Reply:
ya, i am not sure either. But you are right, it should just look like a diff :(